core/src/main/java/org/elasticsearch/search/aggregations/metrics/sum/Sum.java

```diff
@@ -29,4 +33,13 @@
      * The sum.
      */
     double getValue();
+
+    // TODO we could pull common xContent rendering into a static method in the interface
```

Comment:
I tried doing this for the aggregations in this PR in	1048678, but I'm not sure I like it. It avoids some code duplication but makes reading and reasonign about the code much harder in my opinion. Maybe we should only do it when it really saves a lot of duplication (like really long xContent methods), wdyt?